src/android/app/src/main/java/io/github/lime3ds/android/utils/EmulationMenuSettings.kt

```diff
@@ -7,6 +7,7 @@ package io.github.lime3ds.android.utils
 import androidx.drawerlayout.widget.DrawerLayout
 import androidx.preference.PreferenceManager
 import io.github.lime3ds.android.LimeApplication
+import io.github.lime3ds.android.display.PortraitScreenLayout
```

Comment:
Why is this imported when it isn't used?

Reply:
fair question, looks like another case of me just trying to match the io.github.lime3ds.android.display.ScreenLayout file that already existed, like the comment below. They are both imported by this file and not used! I can commit a removal of both imports.